package-lock.json

```diff
@@ -47648,4 +47648,4 @@
       "integrity": "sha512-rVksvsnNCdJ/ohGc6xgPwyN8eheCxsiLM8mxuE/t/mOVqJewPuO1miLpTHQiRgTKCLexL4MeAFVagts7HmNZ2Q=="
     }
   }
-}
+}
```

Comment:
This should fix it all! :D

```suggestion
}

```

Reply:
Krish, what do you mean? Do I have to make some changes to make this pull request close?